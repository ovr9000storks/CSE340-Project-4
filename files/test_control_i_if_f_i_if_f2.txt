
a , b , c, d ;
{

input c;
input d;
c = c + d;
d = c + d;
IF d > c {
	output a;
	output b;
	output c;
	output d;
	IF a > c {
		a = a+1;
		b = b+1;
		output a;
		output b;
	}
	output a;
	output b;
	IF c > a { 
		a = a+2;
		b = b+3;
		output a;
		output b;
	}
	a = a+1;
	output a;
}
input a;
input b;
IF b > a {
	output a;
	output b;
	output c;
	output d;
}
	
 
IF  b > a {
        a = a+b;
	output a;
	IF b < a { 
		output a;
		b = a+b;
		IF b > a {
			output a ;
			output b ;
			IF b < a {
				output a;
				output b;
			}
		}
	}
	a = a+b;
	output a;
} 
a = a*b;
output a;
output b;
IF a > b {
	output a;
	b = 1;
	a = a+b;
	output a;
	output b;
}

}
28 9 2 1 3 8
