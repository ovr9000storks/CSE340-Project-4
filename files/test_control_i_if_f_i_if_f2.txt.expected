0 0 37 46 0 0 2 3 3 2 1 2 3 1 